templates/careers/application/faq.html

```diff
@@ -0,0 +1,300 @@
+{% extends '/careers/base_job-details.html' %}
+
+{% block extra_metatags %}
+  <meta name="robots" content="noindex">
+{% endblock %}
+
+{% block content %}
+ 
+<section class="p-strip is-deep">
+  <div class="u-fixed-width">
+    <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">FAQ</h2>
```

Comment:
This is the first heading on the page, so should be `h1` - and the `h3`s below should be bumped up to `h2`
```suggestion
    <h1 class="p-text--x-small-capitalised u-align-text--x-small-to-default">FAQ</h1>
```